### PR TITLE
Update documentation on allowing users to configure branches to be displayed

### PR DESCRIPTION
**Pull request by amalfra**
https://github.com/Shippable/docsv2/issues/324

http://docs.shippable.com/ci_projects/#dashboard
http://docs.shippable.com/ci_subscriptions/

### CODE REVIEW

File: sources/ci_projects.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ To get to the page for a particular project,
 ## Dashboard
 The Project Dashboard shows you the latest status of all branches of the project. You can also view in progress builds and build history.
 
-The **Summary View** is the default when you navigate to this dashboard. It shows you the latest status for all branches that are built using Shippable. Latest status is defined as the build status for the latest commit build for that branch.
+The **Summary View** is the default when you navigate to this dashboard. It shows you the latest status for all configured branches(see `Dashboard Settings`) that are built using Shippable. Latest status is defined as the build status for the latest commit build/pull request build(as configured in `Dashboard Settings`) for that branch.
```
Comment (abhijitkini) [inline sources/ci_projects.md]: Re-word the above paragraph as:

The **Summary View** is the default view when you navigate to this dashboard. It shows you the latest status for all configured branches that are built using Shippable. Latest status for a build is defined as the build status for the most recent commit build or pull request build for that branch.

Comment (abhijitkini) [inline sources/ci_projects.md]: Move the entire section of **Dashboard Settings** between the History View and the "Manually building a branch"

File: sources/ci_projects.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ NOTE: To view time stamp for a build, hover the mouse over time details
 The **History View** shows the build history across all branches in the project. You can filter this view by commit vs pull request builds, build status, and branch name.
```
Comment (abhijitkini) [inline sources/ci_projects.md]: Move the entire section of **Dashboard Settings** here (between the History View and the "Manually building a branch")

File: sources/ci_subscriptions.md [hunk 3]
```diff
@@ -16,15 +16,15 @@ The `CI` tab of your Subscription page shows you the latest status of all enable
 
 <img src="../images/subscription_dashboard.png" alt="Subscription Dashboard" style="width:800px;"/>
 
-The **Summary View** is the default when you navigate to this dashboard. It shows you the latest status for all enabled projects. Latest status is defined as the build status for the latest commit build on the default branch. If no default branch exists for your project, project status is latest commit build status for the master branch, and if no master branch exists, then the project status is the build status of latest commit build across all branches.
+The **Summary View** is the default when you navigate to this dashboard. It shows you the latest status for all enabled projects. Latest status is defined as the build status for the latest commit run/pull request run across all configured branches(as configured in `Dashboard Settings`). More details on this are in `Dashboard Settings` of our [Projects section](ci_projects.md). By default this setting will be set to your default branch and to show only commit runs.
```
Comment (abhijitkini) [inline sources/ci_subscriptions.md]: Incorporate the following changes:

- latest commit run or pull request run across all configured....
- More details on this in the 'Dashboard Settings for Projects' (Link to the Dashboard settings)
- Your default branch and 'Show only Commit runs' are set as default in the Summary view.

**Comment by abhijitkini** [comment]
@amalfra I have added feedback for your PR. Please review & take further action.

**Comment by manishas** [comment]
We need to re-review this. Please do not make any changes.

**Comment by manishas** [comment]
closing this PR. the feature was misinterpreted and we need to make some changes.

**Comment by amalfra** [comment]
Reopening since the changes are done: https://github.com/Shippable/pm/issues/5950#issuecomment-218069210

File: sources/ci_projects.md [hunk 4]
```diff
@@ -94,7 +102,6 @@ Select the branch you want the status for and pick between the 'Image URL' or 'M
 
 You can use this code to display your code coverage report after code coverage runs. The badge displays appropriate colors depending on the code coverage configuration in the `shippable.yml`. Refer the  [documentation](http://docs.shippable.com/ci_configure/#test_code_coverage) on how to configure code coverage reports.
 
-
 ### Resetting the project
 Resetting a project recreates all webhooks and deployment keys for your project. This should only be done if your project is in an inconsistent state and you need to restore it. Please note that you will need to re-encrypt all environment variables for your project after resetting it.
```
Comment (abhijitkini) [inline sources/ci_projects.md]: Please move the entire section of **Dashboard Settings** to right above the "Clearing Cache". Do include a screen shot of the Dashboard settings.

NOTE: The suggestion above to move Dashboard settings between History view & "Manually building a branch" is an error.

**Comment by abhijitkini** [comment]
@amalfra: Closing this PR without merging. I am opening a new one with your changes incorporated & few more additions.

**Pull request closed without merging**